### ISSUE TITLE
Change colours on FreeREG as per #931

### ISSUE THREAD

**Issue by PatReynolds** [FreeUKGen/MyopicVicar#2050]

**Comment by Captainkirkdawson** [comment]
@PatReynolds  I cannot find the colour schemes for REG (Or the other Apps) anywhere

**Comment by richpomfret** [comment]
@Captainkirkdawson and @AlOneill to meet and discuss next steps for this.

**Comment by Captainkirkdawson** [comment]
Work has been undertaken as part of systemwide test2 contains the current state

**Comment by richpomfret** [comment]
Let's deploy.
